djinn.py

```diff
@@ -123,7 +123,7 @@ async def poll(self,
         self.channels_with_polls.add(channel)
         await channel.send('Wait while I search my boundless library')
 
-        await self.publish_movies(channel=channel, options=options, self.vote_emoji)
+        await messages = self.publish_movies(channel=channel, options=options, self.vote_emoji)
```

Comment:
wrong syntax, should be `messages = await self.publish_movies(channel=channel, options=options, self.vote_emoji)`